boggle-sveltexpress/svelte/src/components/inputs/NumberInput.svelte

```diff
@@ -0,0 +1,162 @@
+<script lang="ts">
+  export let label: string = null;
```

Comment:
label does not need to be set to null, undefined also equals to false.
So you can do `if(label)` instead of `if(label != null)`

Reply:
undefined does not equal false.
`undefined`, `null`, `false`, `0`, `NaN` and "" are all **falsy** statements (see [Falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)).

I can alter the code to check for any falsy value which would also make it ignore empty strings.